src/components/pages/units-page/edit-unit-dialog.tsx

```diff
@@ -55,6 +56,7 @@ interface MusterWindow {
   end: number,
 }
 
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
It looks like this isn't necessary. I'm not getting any errors when removing it.